internal/proto5server/server_readresource_test.go

```diff
@@ -172,6 +174,43 @@ func TestServerReadResource(t *testing.T) {
 				NewState: testEmptyDynamicValue,
 			},
 		},
+		"request-private": {
```

Comment:
We should also create a "response-private" test, similar to the below, but it calls `resp.Private.SetKey()` 👍